### DISCUSSION TITLE
个人信息版块的follow(关注我)文字能否自定义

### DISCUSSION THREAD

**Discussion by nanbowanya** [ppoffice/hexo-theme-icarus#882] [Q&A] [answered]
请问个人信息版块的follow(关注我)文字能否自定义，对应的修改在哪里，如何修改？

**Comment by ppoffice** [comment, marked as answer]
@nanbowan323 最简单的方式是修改中文翻译文件：
https://github.com/ppoffice/hexo-theme-icarus/blob/master/languages/zh-CN.yml#L20

复杂一点的话就要自定义挂件了，这个文档里FAQ有说方法。

**Reply by nanbowanya** [reply]
感谢
